sys/hacl/build.rs

```diff
@@ -190,6 +267,18 @@ fn main() {
     let out_dir = env::var("OUT_DIR").unwrap();
     let out_path = Path::new(&out_dir);
 
+    // Check platform support
+    let platform = Platform {
+        simd128: libcrux_platform::simd128_support(),
+        simd256: libcrux_platform::simd256_support(),
+        aes_ni: libcrux_platform::aes_ni_support(),
+        x25519: libcrux_platform::x25519_support(),
+        bmi2_adx_support: libcrux_platform::bmi2_adx_support(),
+        pmull: libcrux_platform::pmull_support(),
+        adv_simd: libcrux_platform::adv_simd_support(),
+        sha256: libcrux_platform::sha256_support(),
+    };
+
     // Set re-run trigger for all of c
```

Comment:
For some reason we always rebuild (I think because of the copies, ideally we get rid of them).
But to avoid rebuilding every time. Let's remove these two lines and add `println!("cargo:rerun-if-changed=build.rs");` instead.
Alternatively, you could try to build without copying the source files to see if that solves the issue.

Reply:
Fixed!